langchain/chains/llm_bash/demo.py

```diff
@@ -0,0 +1,14 @@
+from dotenv import load_dotenv
```

Comment:
i love this example, but its probably better as a notebook (https://github.com/hwchase17/langchain/tree/master/docs/examples/chains)

Reply:
definitely! it's for my own edification, will remove when I've finished converting the utility :)

Reply:
actually on second thought I'll add it as a notebook 😄